### PR TITLE
[windows] Add system tests for httpjson input

### PR DESCRIPTION
**Pull request by marc-gr**
## What does this PR do?

- Adds system tests for Splunk httpjson inputs
- Fixed sysmon pipeline
- Improved readme

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
~- [ ] If I'm introducing a new feature, I have modified the Kibana version constraint in my package's `manifest.yml` file to point to the latest Elastic stack release (e.g. `^7.13.0`).~

## Related issues

- Relates #961 
- Requires https://github.com/elastic/stream/pull/23 and v0.5.0 to be released

### CODE REVIEW

**Review by P1llus — APPROVED** [review]
LGTM

File: packages/windows/data_stream/forwarded/_dev/test/system/test-default-config.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+input: httpjson
+service: forwarded
+service_notify_signal: SIGHUP
+vars:
+  url: http://{{Hostname}}:{{Port}}/api/v1/logs
+  username: test
+  password: test
```
Comment (andrewkroh) [inline packages/windows/data_stream/forwarded/_dev/test/system/test-default-config.yml:7]: As an enhancement we should make `stream` verify the basic auth.

**Review by andrewkroh — APPROVED** [review]
LGTM

**Pull request merged**